tools/hanabi_table.py

```diff
@@ -131,22 +131,20 @@ def can_disclose_color(self, color):
 
     def disclose_rank(self, player_id, to_whom, rank):
         self.disclosures -= 1
-        count = 0
+        action = HanabiDiscloseRankAction(player_id, to_whom, rank)
         for card in self.hands[to_whom].hand:
             if card.rank == rank:
                 card.disclose_rank()
-                count += 1
-        action = HanabiRankDiscloseAction(player_id, to_whom, rank, count)
+                action.count += 1
```

Comment:
This is maybe a silly thing, but what do you think about making `count` private so it can't be mutated directly, and instead exposing something like an `increment_count` method? To me, that would more clearly indicate how `count` should be used.